src/node_crypto.cc

```diff
@@ -5109,6 +5109,8 @@ EC_POINT* ECDH::BufferToPoint(char* data, size_t len) {
     return nullptr;
   }
 
+  MarkPopErrorOnReturn mark_pop_error_on_return;
```

Comment:
It's not wrong but I'd put this in `ECDH::ComputeSecret()` and `ECDH::SetPublicKey()`, the binding functions that call `ECDH::BufferToPoint()`.

Maybe we should simply make it a rule that binding functions must restore the error stack.